source/MaterialXView/Viewer.cpp

```diff
@@ -519,6 +513,50 @@ void Viewer::createLoadMaterialsInterface(Widget* parent, const std::string& lab
     });
 }
 
+void Viewer::createSaveMaterialsInterface(Widget* parent, const std::string& label)
+{
+    ng::Button* materialButton = new ng::Button(parent, label);
+    materialButton->setIcon(ENTYPO_ICON_SAVE);
+    materialButton->setCallback([this]()
+    {
+        mProcessEvents = false;
+        std::string filename = ng::file_dialog({ { "mtlx", "MaterialX" } }, true);
+
+        //sync document with editor changes
```

Comment:
Minor syntax. Should be space an comment captilized "// Sync ". Same for other comments.